src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -68,6 +82,11 @@ public bool ChildContentRetrieved
         }
 
         /// <summary>
+        /// TODO
+        /// </summary>
+        public IDictionary<string, object> Items { get; }
```

Comment:
Need to have a publicly visible property to create the `TagHelperContext` with the appropriate dictionary. This is essentially the metadata object used to construct all the requirements for running `TagHelper`s.